site/developer/get-started-developer-framework.qmd

```diff
@@ -27,7 +27,7 @@ listing:
     fields: [title, description]
     contents: 
     - ../notebooks/code_samples/customization/customizing_metrics_with_output_templates.ipynb
-    - ../notebooks/code_samples/NLP_and_LLM/prompt_validation_demo.ipynb
+    - ../notebooks/code_samples/nlp_and_llm/prompt_validation_demo.ipynb
     - ../notebooks/code_samples/time_series/tutorial_time_series_forecasting.ipynb
```

Comment:
```suggestion
    - ../notebooks/code_samples/time_series/quickstart_time_series_full_suite.ipynb
```

Reply:
This notebooks has been renamed on the latest from `developer-framework` according to Andres: https://github.com/validmind/developer-framework/blob/main/notebooks/code_samples/time_series/quickstart_time_series_full_suite.ipynb